compiler/dcalc/from_scopelang.ml

```diff
@@ -890,7 +914,7 @@ let translate_rule
                      defined, we add an check "ErrorOnEmpty" here. *)
```

Comment:
If you're sure the error on empty is not needed here then the comment above should be deleted as well.